### ISSUE TITLE
Mevex - Junior Software Developer - Ottawa

### ISSUE THREAD

**Issue by kspaans** [TechnologyMasters/jobs#250]
### Salary Expectation

- negotiable
- Full Time

### Location

- Stittsville (Ottawa), Ontario

## What You'll Do _(Job Description)_

We are looking for a couple junior software developers with a computer
science, computer/software engineering or even electrical engineering
degree if the candidate has an interest/talent in coding.  We would
also consider someone with an equivalent college diploma.  Experience
with industrial automation is a plus but is not a requirement.  We
have a very laid back work environment and a flat corporate structure.
What that means is that you won't necessarily report directly to
anyone specifically but there will be more senior developers to
provide guidance as required. Due to this, being self-motivated is a
required asset.

The position will come with plenty of opportunity to travel worldwide,
if desired.  Our customer facing software mainly consists of C#
applications with a MS SQL Server backend.  We will shortly be delving
into cloud development to make supporting our customers easier and
there are also many other opportunists to work on interesting internal
software projects.

## About Our company

Mevex is a family owned company in Stittsville that makes
linear particle accelerators (industrial radiation
machines/sterilizers).  We are the #1 linear particle accelerator
manufacturer in the world in the market we compete in.  Our software
team currently employs about a dozen people and the company as a whole
is approaching 100 worldwide with satellite offices in France, Belgium
and Thailand.  We recently just acquired a small software company in
Ottawa that works in the sterilization industry (mainly EO and Gamma
controls systems).  We also started the Mevex Conveyor Company in
Brockville and will be working more closely with Fanuc Robots in the
near future.

## How to apply

In summary, we are looking for someone who is young, smart and
motivated who is looking for their first job or an early career
change.  If you think you might be a good fit please
send your resume my way to Jon.  If anyone has any questions about this please let me know and I can provide answers.

---

## Meta

- [x] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [ ] Remote Worldwide
- [ ] Remote Regional
- [ ] Remote OK

**Comment by tripol** [comment]
I'm interested. Sending my application now :)
